channel/consensus_channel/consensus_channel_test.go

```diff
@@ -7,11 +7,49 @@ import (
 
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/google/go-cmp/cmp"
+
 	"github.com/statechannels/go-nitro/channel/state"
-	"github.com/statechannels/go-nitro/internal/testdata"
 	"github.com/statechannels/go-nitro/types"
 )
 
+type actor struct {
+	Address    types.Address
+	PrivateKey []byte
+}
+
+func (a actor) Destination() types.Destination {
+	return types.AddressToDestination(a.Address)
+}
+
+// actors namespaces the actors exported for test consumption
+type actors struct {
+	Alice actor
+	Bob   actor
+	Brian actor
+	Irene actor
+}
+
+// Actors is the endpoint for tests to consume constructed statechannel
+// network participants (public-key secret-key pairs)
+var Actors actors = actors{
```

Comment:
I don't Actors need to be exported.
```suggestion
var actors actors = actors{
```

Reply:
It's removed in https://github.com/statechannels/go-nitro/commit/c3bcfa7c70e994eb3fb1e3b68958f6f632323488 (in another PR)